src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -253,6 +253,62 @@ namespace winrt::TerminalApp::implementation
 
         _UpdateTabWidthMode();
 
+        if (auto app{ winrt::Windows::UI::Xaml::Application::Current().try_as<winrt::TerminalApp::App>() })
+        {
+            if (auto appPrivate{ winrt::get_self<implementation::App>(app) })
+            {
+                // Lazily load the query palette components so that we don't do it on startup.
+                appPrivate->PrepareForAIChat();
```

Comment:
Doing this during `Create` defeats the purpose of lazily loading it. Constructing it during create _also_ defeats the purpose.